Weather/Weather/Scenes/Base/BaseTableViewController.swift

```diff
@@ -50,7 +50,7 @@ extension BaseTableViewController {
         }
     }
 }
-//MARK: - UITableViewDataSource + UITableViewDelegate
+// MARK: - UITableViewDataSource + UITableViewDelegate
 extension BaseTableViewController: UITableViewDataSource, UITableViewDelegate {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
```

Comment:
Nếu number of section bằng 0 không cần khai báo func này

Reply:
numberOfRowsInSection cần phải có khi comform protocol datasource